test/conf/protractor.saucelabs.conf.js

```diff
@@ -196,96 +193,64 @@ config.multiCapabilities = [
 		platform: 'Windows 10',
 		version: 'latest',
 		logName: 'chrome-latest',
-		specs: [basicSpecs],
-		screenResolution,
-		...defaultCapabilities,
-	},
-	{
-		browserName: 'chrome',
-		platform: 'Windows 10',
-		version: 'latest',
-		logName: 'chrome-latest',
-		specs: [saveMethodFolderSpecs],
-		screenResolution,
-		...defaultCapabilities,
-	},
-	{
-		browserName: 'chrome',
-		platform: 'Windows 10',
-		version: 'latest',
-		logName: 'chrome-latest',
-		specs: [checkMethodFolderSpecs],
-		screenResolution,
-		...defaultCapabilities,
-	},
-	{
-		browserName: 'chrome',
-		platform: 'Windows 10',
-		version: 'latest',
-		logName: 'chrome-latest',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	},
 	{
 		browserName: 'firefox',
 		platform: 'Windows 10',
 		version: 'latest',
 		logName: 'Firefox latest',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	},
 	{
 		browserName: 'internet explorer',
 		platform: 'Windows 8.1',
 		version: 'latest',
 		logName: 'IE11',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	},
 	{
 		browserName: 'MicrosoftEdge',
 		platform: 'Windows 10',
 		version: 'latest',
 		logName: 'Microsoft Edge latest',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	},
 	{
 		browserName: 'safari',
 		platform: 'macOS 10.12',
 		version: '11.0',
 		logName: 'SierraSafari11',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	},
 	{
 		browserName: 'safari',
 		platform: 'macOS 10.14',
 		version: 'latest',
 		logName: 'MojaveSafariLatest',
-		specs: [deskSpecs],
-		screenResolution,
-		...defaultCapabilities,
+		screenResolution: SCREEN_RESOLUTON,
+		...DEFAULT_CAPABILITIES,
 	}
 ];
 
 config.plugins = [
 	{
-		path: join(process.cwd(), './build/index.js'),
+		path: resolve(process.cwd(), './build/index.js'),
 		options: {
-			baselineFolder: join(process.cwd(), './test/sauceLabsBaseline/'),
-			debug: true,
-			formatImageName: `{tag}-{logName}-{width}x{height}`,
-			screenshotPath: join(process.cwd(), '.tmp/'),
+			baselineFolder: resolve(process.cwd(), './test/images/baseline'),
+			screenshotPath: resolve(process.cwd(), './test/images/'),
+			formatImageName: '{tag}-{logName}-{width}x{height}',
+			debug: false,
 			savePerInstance: true,
 			autoSaveBaseline: true,
 			blockOutStatusBar: true,
 			blockOutToolBar: true,
+			clearRuntimeFolder: true
```

Comment:
I need to remove this functionality from the module. The plugin architecture of Protractor is that it will be triggered per instance instead of being in a onBefore / onComplete. Meaning it will remove all folders for each instance making this break for the slower ones.